src/components/Todos.tsx

```diff
@@ -1,22 +1,37 @@
 import { useSelector } from 'react-redux';
 
+import { filters, TodosState } from '../store/store';
 import TodoItem from './TodoItem';
-import { TodosState } from '../store/store';
+import FilterTodo from './FilterTodo';
 
 import classes from './Todos.module.css';
 
 const Todos = () => {
   const items = useSelector((state: TodosState) => state.todos);
+  const filter = useSelector((state: TodosState) => state.filterBy);
 
   let content = <p className={classes.message}>No todos yet</p>;
 
+  const filterTodo = () => {
+    if (filter === filters.COMPLETED) {

```

Comment:
You can use switch/case and maybe useMemo, to avoid recalculating items when filter doesn't change